rfcs/0000-policy_target_scope.md

```diff
@@ -0,0 +1,228 @@
+be affected by the merge of the two policies:
+
+```yaml
+enabled: true
+rps: 500
+```
+
+>**Note:** The status block of a policy should reflect the merged state of a policy as per other
+>policies at that level of the hierarchy.
+
+Any other `HTTPRoute` attached to the `my-gw` would still use the 42 rps limit. If we now delete the
+`FakeRLPolicy` attached to `my-gw`, the resulting policy would look like this:
+
+```yaml
+enabled: true
+rps: 500
+```
+
+Which would effectively disable all rate limiting traffic to the `foo-route` and ultimately the `foo`
+`Backend`.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+So there are a couple of things to consider here:
+
+- Being able to clearly identify and merge `HTTPRouteMatch`es from the `HTTPRoute` and apply
+  accordingly to "lower down" resources
+- What about two `Gateway`s having different policies attached to them? (see [Unresolved
+  questions](#unresolved-questions) below)
+
+This is the technical portion of the RFC. Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the feature would be implemented.
+- How error would be reported to the users.
+- Corner cases are dissected by example.
+
+The section should return to the examples given in the previous section, and explain more fully how
+the detailed proposal makes those examples work.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- The use cases this approach doesn't address… 
+- Proliferation of `HTTPRoute` rather than finer grained control at the policy level or at the
+  `HTTPRouteRule` level
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- [`HTTPRouteFilter`](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPRouteFilter)
+  would be an alternative and provide finer grained control over how a policy applies, i.e. at the
+  `HTTPRouteRule` level, or even on a per `HTTPBackendRef` within an `HTTPRouteRule`. But today there
+  is no way (that I know of) for us to add a `HTTPRouteFilterType` to the list of supported
+  `HTTPRouteFilter` by a Gateway API implementor. 
+
+# Prior art
+[prior-art]: #prior-art
+
+I don't know of prior art per se, _but_ the envoy gateway project, if only for rate limiting, seems to
+[avoid policy attachment](https://github.com/envoyproxy/gateway/issues/670#issuecomment-1299114982),
+but they are an implementor and as such can extend the
+[`HTTPRouteFilter`](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPRouteFilter)
+types.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- How do we deal with multiple `Gateway` having different set of policies attached to them
+  (semantically or atomically?)?
+  - Can we inject the proper config in each of them? Only if all is applicable? What if they are not?
```

Comment:
I do not follow this. 

Yes, we can inject proper config to each gateway. So, 
RLP A -> Gateway A

RLP B -> Gateway B